public/assets/css/general.css

```diff
@@ -516,18 +516,23 @@ table {
 	display: block;
 	width: 100%;
 }
-
-#video-player {
+#video-player, .showlite-asset {
+    padding: 6px;
+    border: 1px solid #555;
     background-color: #fff;
+}
+#video-player {
     cursor: pointer;
-    margin: 0 auto;
-    margin-top: 2%;
-    border: 1px solid #d0d0d0;
-    padding:6px;
 }
 
 .showlite-asset {
     max-width: 50%;
+    margin: 0 auto;
+    margin-top: 2%;
+    padding: 6px;
+    border: 1px solid #a2a2a2;
+//Optional Box Shadow - Looks better on a darker than white background.
```

Comment:
I'm not sure about the box shadow in general, but I do think it works better on `#video-player` than on regular `showlite-asset` assets (e.g. pictures).  How about moving it to #video-player.